tests/Doctrine/Tests/ORM/Functional/EntityRepositoryTest.php

```diff
@@ -655,17 +653,6 @@ public function testDefaultRepositoryClassName(): void
         self::assertEquals($this->_em->getConfiguration()->getDefaultRepositoryClassName(), EntityRepository::class);
     }
 
-    /**
-     * @group DDC-753
-     */
-    public function testSetDefaultRepositoryInvalidClassError(): void
```

Comment:
Does this test need to be removed?

Reply:
It was redundant. `ConfigurationTest` literally tests the same thing.